Description: Makes the build user invariant to improve the reproducibility (it appears in the interval VM version in libjvm.so)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/make/autoconf/basics.m4
+++ b/make/autoconf/basics.m4
@@ -658,7 +658,7 @@ AC_DEFUN_ONCE([BASIC_SETUP_PATHS],
 
   # Setup username (for use in adhoc version strings etc)
   # Outer [ ] to quote m4.
-  [ USERNAME=`$ECHO "$USER" | $TR -d -c '[a-z][A-Z][0-9]'` ]
+  [ USERNAME="unknown" ]
   AC_SUBST(USERNAME)
 ])
 
